clients/flash/air-client/src/org/bigbluebutton/air/screenshare/model/IScreenshareModel.as

```diff
@@ -0,0 +1,43 @@
+package org.bigbluebutton.air.screenshare.model {
+	import org.osflash.signals.ISignal;
+	
+	public interface IScreenshareModel {
+		function get screenshareStreamStartedSignal():ISignal;
+		function get screenshareStreamStoppedSignal():ISignal;
+		
+		function get isSharing():Boolean;
+		
+		function get width():int;
+		
+		function set width(w:int):void;

```

Comment:
I think all of these setters can be removed. I don't see them actually used anywhere outside of the class.